### PR TITLE
LBT Standardizer

### PR DESCRIPTION
**Pull request by arihani2**
Me and Dino are going to take a look at the code tomorrow to figure out what's going on with the astrometry.net stuff and see if we can fix it internally.

### CODE REVIEW

**Comment by arihani2** [comment]
[Large Binocular Telescope Standardizer](https://trello.com/c/01raIeDu/50-large-binocular-telescope-standardizer)

File: trail/upload/process_uploads/standardizers/lbt_standardizer.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+from datetime import datetime, timedelta, timezone
+
+from upload.process_uploads.header_standardizer import HeaderStandardizer
+from upload.models import Metadata
+
+
+__all__ = ["LbtStandardizer", ]
+
+
+class LbtStandardizer(HeaderStandardizer):
```
Comment (mrawls) [inline trail/upload/process_uploads/standardizers/lbt_standardizer.py:11]: Please add a class docstring and also function docstrings, ideally following the guide here. It's based on numpydoc and has easy-to-follow examples. https://developer.lsst.io/python/numpydoc.html

**Review by mrawls — APPROVED** [review]
This looks great, assuming you made sure it works on a real image or two 😄. @DinoBektesevic do we have a trello or other ticket for adding tests for each of the new standardizers? I don't think the work needs to be done on this PR, but we shouldn't write a bunch of fragile instrument-specific standardizers and not have some kind of testing in place for them. Happy to discuss at our meeting.

**Review by DinoBektesevic — APPROVED** [review]
Looks nice! No comments, you can push the big green button.

**Pull request merged**